src/App/components/BookSelector.tsx

```diff
@@ -28,12 +28,21 @@ export default function BookSelector(props: any): JSX.Element | null {
                 m={1}
                 key={book.name}
                 intent={book.isSelected ? Intent.PRIMARY : undefined}
-                onClick={(): void => { project.setActiveBook(book.name); }}
+                onClick={(): void => {
+                  project.setActiveBook(book.name);
+                }}
                 active={project.activeBookName === book.name}
               >
                 {book.name}
                 {book.isSelected && project.bookSelection.length > 1 && (
-                  <Tag position="absolute" zIndex={2} right="-10px" top="-10px"round intent={Intent.SUCCESS}>
+                  <Tag
+                    position="absolute"
+                    style={{ zIndex: 2 }}
```

Comment:
Note: the only non-formatting change in this whole branch is the replacement of `zIndex={2}` with `style={{zIndex:2}}` right here.

Reply:
I'm concerned that there doesn't seem to be a corresponding change in FileSelector.tsx. Can somebody who is familiar with the issue please explain what to look for?